builtin/logical/pki/path_issue_sign.go

```diff
@@ -167,18 +197,33 @@ func buildPathSign(b *backend, pattern string) *framework.Path {
 
 func pathIssuerSignVerbatim(b *backend) *framework.Path {
 	pattern := "issuer/" + framework.GenericNameRegex(issuerRefParam) + "/sign-verbatim" + framework.OptionalParamRegex("role")
-	return buildPathIssuerSignVerbatim(b, pattern)
+
+	displayAttrs := &framework.DisplayAttributes{
+		OperationPrefix: operationPrefixPKIIssuer,
+		OperationVerb:   "sign",
+		OperationSuffix: "verbatim|verbatim-for-role",
```

Comment:
One last nit, take it or leave it: could these be `with-role` rather than `for-role`? I think that makes a little more sense, IMO.

Reply:
Good point, will fix, thanks!